packages/web/src/utils/pool-utils.ts

```diff
@@ -20,7 +21,7 @@ export function makePoolPath(
   }
   const tokenAPath = tokenA.wrappedPath || tokenA.path || "";
   const tokenBPath = tokenB.wrappedPath || tokenB.path || "";
-  return [tokenAPath, tokenBPath].sort().join(":") + ":" + SwapFeeTierInfoMap[swapFeeTier].fee;
+  return [tokenAPath, tokenBPath].sort(sortTokenPaths).join(":") + ":" + SwapFeeTierInfoMap[swapFeeTier].fee;
```

Comment:
How about use `join()` consistently to concatenate all elements at one?

```suggestion
  return [...[tokenAPath, tokenBPath].sort(sortTokenPaths), SwapFeeTierInfoMap[swapFeeTier].fee].join(":");
```

Reply:
We've modified the code to make it consistent.